src/platform_impl/windows/window.rs

```diff
@@ -157,10 +161,12 @@ impl Window {
 
     #[inline]
     pub fn inner_size(&self) -> PhysicalSize<u32> {
-        let mut rect: RECT = unsafe { mem::zeroed() };
-        if unsafe { winuser::GetClientRect(self.window.0, &mut rect) } == 0 {
-            panic!("Unexpected GetClientRect failure: please report this error to https://github.com/rust-windowing/winit")
-        }
+        let rect = self.window_state.lock()
+            .saved_inner_rect
+            .unwrap_or_else( ||
+                util::get_client_rect(self.window.0).expect("Unexpected GetClientRct failure: please report this error to https://github.com/rust-windowing/winit")
```

Comment:
```suggestion
                util::get_client_rect(self.window.0).expect("Unexpected GetClientRect failure: please report this error to https://github.com/rust-windowing/winit")
```

Reply:
Thank you for noticing me typos! I fixed them.